ios/RNAdvertisingId.m

```diff
@@ -8,6 +8,11 @@
 
 @interface RCT_EXTERN_MODULE(RNAdvertisingId, NSObject)
 
-RCT_EXTERN_METHOD(getAdvertisingId:(RCTResponseSenderBlock)callback)
+RCT_EXTERN_METHOD(getAdvertisingId:(RCTPromiseResolveBlock)resolve rejecter: (RCTPromiseRejectBlock)reject)
+
++ (BOOL)requiresMainQueueSetup
+{
+  return NO;
+}
```

Comment:
If we don't explicitly disable main queue setup in this manner react native throws warnings about initializing on the main thread. This warning would appear for anyone using the package.